src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPicker.cs

```diff
@@ -504,6 +515,8 @@ private new void UpdateVisualState(bool useTransitions)
 			VisualStateManager.GoToState(this, isAlphaEnabled && this.IsAlphaTextInputVisible ? "AlphaTextInputVisible" : "AlphaTextInputCollapsed", useTransitions);
 			VisualStateManager.GoToState(this, this.IsHexInputVisible ? "HexInputVisible" : "HexInputCollapsed", useTransitions);
 			VisualStateManager.GoToState(this, isAlphaEnabled ? "AlphaEnabled" : "AlphaDisabled", useTransitions);
+			VisualStateManager.GoToState(this, isVerticalOrientation ? "Vertical" : "Horizontal", useTransitions);
+
```

Comment:
nit: I believe blank lines at the end are considered formatting errors by most. I didn't copy over formatting errors from upstream.